pinot-server/src/main/java/org/apache/pinot/server/starter/helix/SegmentReloadStatusValue.java

```diff
@@ -22,10 +22,16 @@
 public class SegmentReloadStatusValue {
   private final long _totalSegmentCount;
   private final long _successCount;
+  private final Long _failureCount;
```

Comment:
can we use `long` here instead of `Long`. there is usually some overhead in Java to unbox. You can perhaps initialize this with `-1L`?